pkg/cmd/admin/diagnostics/diagnostics.go

```diff
@@ -80,13 +82,23 @@ you will receive an error if they are not found. For example:
 Diagnostics may be individually run by passing diagnostic name as arguments.
 The available diagnostic names are:
 %[2]s
+`
 
-NOTE: This is a beta version of diagnostics and may still evolve in a
-different direction.
+	deprecationNotice = `use "oadm %[1]s" to run diagnostics instead.
```

Comment:
This line is sufficient.  If you'd like to add something about removal that's fine, but no eye catchers.